udata/settings.py

```diff
@@ -419,6 +416,7 @@ class Defaults(object):
 
     FIXTURE_DATASET_SLUGS = []
     PUBLISH_ON_RESOURCE_EVENTS = False
+    RESOURCES_ANALYSER_URI = 'http://localhost:8000'
```

Comment:
I'm not 100% convinced by this name. It feels like we would call this to get an analysis, whereas our first intention is to notify it from a ressource event.